### DISCUSSION TITLE
API & Webhooks: December Community Check-In ⛄

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#40641] [Apps, API and Webhooks]
Hey folks, reporting in for our December API and Webhooks Check-In ✅

## 🧠​​ Pros in the Community
**Big kudos to these superstar community contributors 👏**
- Shoutout to @LangLangBart for showing us [how to get the date & time of requested reviews of a pull request using GraphQL](https://github.com/orgs/community/discussions/38464) 📚
- Kudos to @david-engelmann for helping a community member [troubleshoot an issue where their GraphQL query was returning a null value](https://github.com/orgs/community/discussions/32478) 🕵️‍♂️
- Sometimes the answer comes from the original poster themselves. Thank you @manuelbuil for circling back [on your Discussion post](https://github.com/orgs/community/discussions/39745) so that you could document the answer for users in the future who run into a similar issue ♻️

## ❓​​Got your Thinking Cap On?
**Here are a few questions from last month that could still use an answer:**
- [REST API for Getting the license for a repository…](https://github.com/orgs/community/discussions/38797)
- [403 when accessing private package](https://github.com/orgs/community/discussions/38856)
- [trouble with github graphQL API authentication](https://github.com/orgs/community/discussions/39200)
- [list repo issue templates via the API without having "repo" access](https://github.com/orgs/community/discussions/38882)
- [How can I review all Personal Access Tokens (classic and fine-grained) in my organisation?](https://github.com/orgs/community/discussions/39421)

## 🗞️What’s New?
**Have you heard? We’ve introduced calendar-based versioning for our REST API 🗣️**
- You can read more in our announcement here: [To infinity and beyond: enabling the future of GitHub’s REST API with API versioning](https://github.blog/2022-11-28-to-infinity-and-beyond-enabling-the-future-of-githubs-rest-api-with-api-versioning/)
- **TL;DR:** The GitHub REST API is now versioned. The API version name is based on the date when the API version was released. For example, the API version 2022-11-28 was released on Mon, 28 Nov 2022. Any breaking changes will be released in a new API version. For reference, “breaking changes” refer to changes that can potentially break an integration. You can find a list of breaking changes in our documentation here: [About API versioning](https://docs.github.com/en/rest/overview/api-versions?apiVersion=2022-11-28#about-api-versioning).

**Join the public beta of webhook forwarding in the GitHub CLI**
- You can read more about it in our Community post here: [[Feedback Tracking] Fine-Grained Personal Access Tokens](https://github.com/orgs/community/discussions/36441) 
- **TL;DR:** Fine-grained personal access tokens give developers granular control over the permissions and repository access they grant to a personal access token. Organization administrators are in control too, with approval policies and full visibility for tokens that access organization resources.
- Participants in the beta program should share their feedback and questions with the GitHub team here: [Feedback Tracking Webhook forwarding in the GitHub CLI](https://github.com/orgs/community/discussions/38262) 

## 💡Inspiration Station
**Fresh content to give you new ideas!**
- **Interested in learning about GitHub Copilot?** Check out @blackgirlbytes Community post [An Introduction to GitHub Copilot](https://github.com/orgs/community/discussions/39086).
- **New to open source?** Here are some resources to help you get started:
  - [All In for Students](https://allinopensource.org/access/) is an online program for tech students from underrepresented backgrounds that provides open source education, training, and internship opportunities.
  - [Here’s how to find projects that need help so you can start making impactful contributions today](https://github.com/collections/choosing-projects) 🏗️
  - Check out the article [Hacks to Help Open Source Leadership Support Inclusion
](https://thenewstack.io/hacks-to-help-open-source-leadership-support-inclusion/), which includes contributions from @demetris11, our Senior Director of Diversity and Inclusion Strategy at GitHub
- **Public APIs:** Ever needed to find a free API for your next project, website, or app? Check out the [Public APIs Repository](https://github.com/public-apis-dev/public-apis) 📃

## 🧹Housekeeping
A very big thank you to everyone in the community who actively contributes thoughts and answers to the questions that come into the API and Webhooks category. For community members who post questions, marking responses that solves your question as the answer goes a long way.

When you mark a comment in the Discussion as an answer, GitHub will highlight the comment and replies to the comment to help other community members quickly find the answer as well. You can learn more about marking a response as the answer to your Community post [here](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer).  

— 

Have any projects you built that you would like to share? Link them below! Cheers 😄

**Comment by kaiykay** [comment]
How can this repo be treated as normal root or as it supports super user

**Reply by Clzango** [reply]
hello,sorry , but can you help me with flask?my email :czango45@gmail.com, i need a help

**Reply by Raza1234567809876543** [reply]
yes 

[Action Camera Protector](https://www.actioncameraprotector.us)

**Comment by Cuzzo30** [comment]
#26256

**Comment by Masuan** [comment]
0115365147

**Comment by ArslanArdavic** [comment]
> Hey folks, reporting in for our December API and Webhooks Check-In white_check_mark
> 
> ## ​​ Pros in the Community
> **Big kudos to these superstar community contributors clap**
> 
> * Shoutout to @LangLangBart for showing us [how to get the date & time of requested reviews of a pull request using GraphQL](https://github.com/orgs/community/discussions/38464) books
> * Kudos to @david-engelmann for helping a community member [troubleshoot an issue where their GraphQL query was returning a null value](https://github.com/orgs/community/discussions/32478) male_detective
> * Sometimes the answer comes from the original poster themselves. Thank you @manuelbuil for circling back [on your Discussion post](https://github.com/orgs/community/discussions/39745) so that you could document the answer for users in the future who run into a similar issue recycle
> 
> ## ​​Got your Thinking Cap On?
> **Here are a few questions from last month that could still use an answer:**
> 
> * [REST API for Getting the license for a repository…](https://github.com/orgs/community/discussions/38797)
> * [403 when accessing private package](https://github.com/orgs/community/discussions/38856)
> * [trouble with github graphQL API authentication](https://github.com/orgs/community/discussions/39200)
> * [list repo issue templates via the API without having "repo" access](https://github.com/orgs/community/discussions/38882)
> * [How can I review all Personal Access Tokens (classic and fine-grained) in my organisation?](https://github.com/orgs/community/discussions/39421)
> 
> ## What’s New?
> **Have you heard? We’ve introduced calendar-based versioning for our REST API speaking_head**
> 
> * You can read more in our announcement here: [To infinity and beyond: enabling the future of GitHub’s REST API with API versioning](https://github.blog/2022-11-28-to-infinity-and-beyond-enabling-the-future-of-githubs-rest-api-with-api-versioning/)
> * **TL;DR:** The GitHub REST API is now versioned. The API version name is based on the date when the API version was released. For example, the API version 2022-11-28 was released on Mon, 28 Nov 2022. Any breaking changes will be released in a new API version. For reference, “breaking changes” refer to changes that can potentially break an integration. You can find a list of breaking changes in our documentation here: [About API versioning](https://docs.github.com/en/rest/overview/api-versions?apiVersion=2022-11-28#about-api-versioning).
> 
> **Join the public beta of webhook forwarding in the GitHub CLI**
> 
> * You can read more about it in our Community post here: [[Feedback Tracking] Fine-Grained Personal Access Tokens](https://github.com/orgs/community/discussions/36441)
> * **TL;DR:** Fine-grained personal access tokens give developers granular control over the permissions and repository access they grant to a personal access token. Organization administrators are in control too, with approval policies and full visibility for tokens that access organization resources.
> * Participants in the beta program should share their feedback and questions with the GitHub team here: [Feedback Tracking Webhook forwarding in the GitHub CLI](https://github.com/orgs/community/discussions/38262)
> 
> ## Inspiration Station
> **Fresh content to give you new ideas!**
> 
> * **Interested in learning about GitHub Copilot?** Check out @blackgirlbytes Community post [An Introduction to GitHub Copilot](https://github.com/orgs/community/discussions/39086).
> * **New to open source?** Here are some resources to help you get started:
>   
>   * [All In for Students](https://allinopensource.org/access/) is an online program for tech students from underrepresented backgrounds that provides open source education, training, and internship opportunities.
>   * [Here’s how to find projects that need help so you can start making impactful contributions today](https://github.com/collections/choosing-projects) building_construction
>   * Check out the article [Hacks to Help Open Source Leadership Support Inclusion
>     ](https://thenewstack.io/hacks-to-help-open-source-leadership-support-inclusion/), which includes contributions from @demetris11, our Senior Director of Diversity and Inclusion Strategy at GitHub
> * **Public APIs:** Ever needed to find a free API for your next project, website, or app? Check out the [Public APIs Repository](https://github.com/public-apis-dev/public-apis) page_with_curl
> 
> ## Housekeeping
> A very big thank you to everyone in the community who actively contributes thoughts and answers to the questions that come into the API and Webhooks category. For community members who post questions, marking responses that solves your question as the answer goes a long way.
> 
> When you mark a comment in the Discussion as an answer, GitHub will highlight the comment and replies to the comment to help other community members quickly find the answer as well. You can learn more about marking a response as the answer to your Community post [here](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer).
> 
> —
> 
> Have any projects you built that you would like to share? Link them below! Cheers smile

@ArslanArdavic

**Comment by ghostinhershell** [comment]
Hey Community 👋 

Missed out on our last community check-in? You can check that out here: [API and Webhooks: October Community Check-in 👻](https://github.com/orgs/community/discussions/34998)
